proms-resources/frontend/src/main/frontend/src/index.jsx

```diff
@@ -77,6 +100,17 @@ function PromsHomepage (props) {
       </Toolbar>
     </AppBar>
     <QuestionnaireSet id={promId} subject={subject} />
+    <ResponsiveDialog
+      title="Terms of Use"
+      open={showTou}
+      withCloseButton
+      width="lg"
+      onClose={() => {setShowTou(false);}}
+    >
+     <DialogContent dividers>
+       <FormattedText variant="caption">{tou.text}</FormattedText>
```

Comment:
`caption` is too small, let's use the default instead.
```suggestion
       <FormattedText>{tou.text}</FormattedText>
```